tests/directives/test_item_directive.py

```
@@ -1,17 +1,20 @@
 from sphinx.application import Sphinx
 from sphinx.builders.latex import LaTeXBuilder
 from sphinx.builders.html import StandaloneHTMLBuilder
 from sphinx.environment import BuildEnvironment
 from sphinx.errors import NoUri
 
-from mlx.traceability.directives.item_directive import Item as dut
+from mlx.traceability.directives.item_directive import Item as dut, ItemDirective
 from mlx.traceability.traceable_collection import TraceableCollection
 from mlx.traceability.traceable_item import TraceableItem
+from mlx.traceability.traceable_attribute import TraceableAttribute
+from mlx.traceability.traceability_exception import TraceabilityException
```

Flake8 (7.3.0)

[error] 'mlx.traceability.traceability_exception.TraceabilityException' imported but unused

(F401)